### ISSUE TITLE
'print_escaped' not defined.

### ISSUE THREAD

**Issue by joa-quim** [JuliaGraphics/Tk.jl#140]
```
julia> using Tk
INFO: Recompiling stale cache file c:\j\.julia\lib\v0.6\Cairo.ji for module Cairo.
ERROR: LoadError: LoadError: UndefVarError: print_escaped not defined
Stacktrace:
 [1] init() at c:\j\.julia\v0.6\Tk\src\tkwidget.jl:49
 [2] include_from_node1(::String) at .\loading.jl:569
 [3] include(::String) at .\sysimg.jl:14
 [4] include_from_node1(::String) at .\loading.jl:569
 [5] eval(::Module, ::Any) at .\boot.jl:235
 [6] _require(::Symbol) at .\loading.jl:483
 [7] require(::Symbol) at .\loading.jl:398
while loading c:\j\.julia\v0.6\Tk\src\tkwidget.jl, in expression starting on line 457
while loading c:\j\.julia\v0.6\Tk\src\Tk.jl, in expression starting on line 37
```

**Comment by ararslan** [comment]
Looks like it was deprecated in https://github.com/JuliaLang/julia/pull/16603 (`0.5.0-dev+4354`) in favor of `escape_string`. Since this package requires 0.5 at a minimum, it should be safe to just replace `print_escaped` with `escape_string`.

**Comment by paulobuchsbaum** [comment]
My computer run Windows. I did the following:

In `c:\Users\...\.julia\v0.6\Tk\src` there is a source `tkwidget.jl` written in Julia. There do the follow replacements.

1) Replace `print_escaped` (depreciated) by `escape_string`. 
2) Replace `takebuf_string(b)` (depreciated) by `String(take!(b))` where b is any parameter. 

After that `Winston` works like a charm.

Example:

```
using Winston
plot(rand(100))
```

**Comment by rafaqz** [comment]
@paulobuchsbaum how about a pull request fixing that? we just had the same problem

**Comment by rafaqz** [comment]
Seems there already are 2 pull requests for this

https://github.com/JuliaGraphics/Tk.jl/pull/139

**Comment by aviks** [comment]
Fixed by #147
